syntax = "proto3";

package notification;

service NotificationService {
  rpc SendNotification (NotificationRequest) returns (NotificationResponse);
}

message NotificationRequest {
  string user_id = 1;
  string message = 2;
}

message NotificationResponse {
  string status = 1;
}
